app/src/main/java/org/fossasia/openevent/app/data/attendee/Attendee.java

```diff
@@ -42,7 +42,7 @@
 
     @Id(LongIdHandler.class)
     @PrimaryKey
-    public long id;
+    public Long id;
```

Comment:
No need

Reply:
@iamareebjamal  In the Adapter, ItemsTheSame() method throws an error `long cannot be dereferenced`. Therefore I had to change `long` to `Long` in model.

Reply:
Use ==